### ISSUE TITLE
Full scan of auction house is broken or blocked

### ISSUE THREAD

**Issue by Morant503** [Ascension-Addons/Auctionator#8]
### Description

`QueryAuctionItems ("", nil, nil, 0, 0, 0, 0, nil, 0, true);`

Polls the server for a list of all auctions and sets values for "numBatchAuctions" and "totalAuctions" . The value set for totalAuctions is correct each time and varies as it should. The value returned for numBatchAuctions is always 3000.

If my understanding of this process is correct the totalAuctions is arranged in pages (batches) of 50 item each. The expected value set for numBatchAuctions is totalAuctions divided by 50 for a total number of item pages. 

IE -  totalAuctions = 47,486 than numBatchAuctions = 950 . (36 auctions in the last page.)

In reality though, the value returned for numBatchAuctions is consistently 3000. Each and every time. Either I am correct in assuming this is a limitation imposed specifically by Project Ascension to disallow a full and complete dataset of the auction house, or something has been changed in how the data is delivered when the event "AUCTION_ITEM_LIST_UPDATE" is fired from the server.

If I am correct and this limitation is intentional and there is not a way for a player to be able to view the auction house contents as a whole outside of the in-game GUI. I would like to understand why the restriction is needed and then to begin the conversation of what can be developed to allow a full data package to arrive that can be parsed into useful information for the player.

If I have something wrong, thank you in advance for the guidance.

### Realm

Seasonal

### Tested with only this addon

- [X] Yes
- [ ] No

### Lua Error

No.

### Reproduction Steps

Click Full Scan button

**Comment by Morant503** [comment]
Tested output on Area 52 server:
`/run local numBatchAuctions, totalAuctions = GetNumAuctionItems("list"); DEFAULT_CHAT_FRAME:AddMessage("Total Auctions: " .. totalAuctions .. " | Num Batch Auctions: " .. numBatchAuctions)`
 
It returns correctly for the two scans I was able to make in 30 minutes. 

EDIT: I just noticed that after both scans: 

totalAuctions = 129,728   numBatchAuctions = 55,000  (5712 items)

This tells me that the server is limiting the entire batch to 55,000 auctions, roughly half.

Does this make it impossible to get a complete shot of the Auction House?
